### DISCUSSION TITLE
What is the function to get the dimensions of a matrix?

### DISCUSSION THREAD

**Discussion by ISibboI** [dimforge/nalgebra#1340] [Q&A] [answered]
Say I have a dynamic matrix. What is the function that gives me the number of rows and number of columns in this matrix?

**Comment by sebcrozet** [comment, marked as answer]
Hi! Its `matrix.nrows()`, `matrix.ncols()`.

**Reply by ISibboI** [reply]
Great, thanks! I was unable to find that on the docs, so maybe it could be mentioned? All other things such as creation, operations, views, and indexing are explained very nicely already.
